src/commands/create.js

```diff
@@ -82,6 +83,14 @@ class CreateCommand extends ApifyCommand {
 
         const packageJsonPath = path.join(actFolderDir, 'package.json');
         const requirementsTxtPath = path.join(actFolderDir, 'requirements.txt');
+        const readmePath = path.join(actFolderDir, 'README.md');
+
+        // Add localReadmeSuffix which is fetched from manifest to README.md
+        // The suffix contains local development instructions
+        if (fs.existsSync(readmePath)) {
```

Comment:
Let's append the readme suffix only if it's not `false`-y, otherwise you could end up with `undefined` in the readme.
```suggestion
        if (localReadmeSuffix && fs.existsSync(readmePath)) {
```